beluga/include/beluga/algorithm/spatial_hash.hpp

```diff
@@ -196,6 +196,61 @@ class spatial_hash<Sophus::SE2d, void> {
   spatial_hash<std::tuple<double, double, double>> underlying_hasher_{{1., 1., 1.}};
 };
 
+/**
+ * Specialization for Sophus::SE3d. Will calculate the spatial hash based on the translation and rotation (expressed in
+ * Euler Angles).
```

Comment:
@serraramiro1 which convention for Euler angles? There are many.

Reply:
Added a comment